src/locales/ar.json

```diff
@@ -73,5 +73,11 @@
         }
       ]
     }
+  },
+  "contact": {
+    "contactUsHeaderSection": {
```

Comment:
contact.headerSection is fine here because the parent part already says contact

Reply:
Thanks for your review, I have changed the key name to "headerSection". PTAL